cognee/modules/data/processing/document_types/Document.py

```
@@ -1,12 +1,18 @@
-    type: str
     name: str
     raw_data_location: str
     metadata_id: UUID
     mime_type: str
+    _metadata: dict = {
+        "type": "Document"
+    }
 
     def read(self, chunk_size: int) -> str:
         pass
+    def read(self, chunk_size: int) -> str:
```

Ruff (0.8.2)

Redefinition of unused `read` from line 15

(F811)